src/vehicle.cpp

```diff
@@ -6154,6 +6158,20 @@ bool vehicle_part::can_reload( const itype_id &obj ) const
     return false;
 }
 
+bool vehicle_part::fill_with( item &liquid, long qty )
+{
+    if( liquid.active ) {
+        // cannot refill using active liquids (those that rot) due to #18570
```

Comment:
Would be better if the comment was informative without having to loop up the issue.
Something like "Because active item processing doesn't process vehicle tanks' contents".